### PR TITLE
Store sign method in sign session

### PR DESCRIPTION
**Pull request by FredrikAreschoug**
In we need to know the sign method for noway.

Changes:
* Store sign method
* Fallbacks to old way of determining sign method

### CODE REVIEW

File: src/main/kotlin/com/hedvig/underwriter/model/SignSessionDao.kt [hunk 1]
```diff
@@ -11,14 +12,14 @@ interface SignSessionDao {
     @SqlUpdate(
         """
             INSERT INTO sign_sessions (
-                id, created_at
+                id, signMethod, created_at
```
Comment (bystam) [inline src/main/kotlin/com/hedvig/underwriter/model/SignSessionDao.kt]: Mixed camel and snake case?

Comment (FredrikAreschoug) [reply src/main/kotlin/com/hedvig/underwriter/model/SignSessionDao.kt]: nice catch :slightly_smiling_face:

**Comment by FredrikAreschoug** [comment]
Looking at this some more this is not the best sulotion

**Comment by FredrikAreschoug** [comment]
Looking at this some more this is probably not the best solution :slightly_smiling_face:

**Pull request closed without merging**